var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -189,6 +189,9 @@ def cmake_args(self):
                 ]
             )
 
+        with when("+cuda @:0.7.1"):
+            args.extend(self.define_from_variant("HIOP_USE_RESOLVE", "cusolver_lu"))
```

Comment:
Maybe there is a cleaner way to do this... It's functional and I am unsure if this was even necessary to resolve the bug we were seeing in the GitHub pipelines, but seems like a sensible change to remove additional whitespace in CMake line.